### PR TITLE
Update UG to specify behaviour of UI

### PR DESCRIPTION
**Pull request by Javiery3889**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -48,10 +48,10 @@ FINDvisor enables financial advisors to quickly save and retrieve the following
...
-3. **Person List**: The list of persons stored in FINDvisor. Note that this list will be display the latest list when a command to update the list is **executed successfully**.
-4. **Today's Meeting List**: The list of persons who have a meeting on the current date of your computer when running the application (if any).
+1. *Command Box*: Commands are entered in this box and executed when the user presses the Enter key. The command entered will be cleared when a command is executed successfully.
+2. *Command Result Box*: The feedback for command will be displayed, indicating if command executed was successful. An error message will be displayed if the command was unsuccessful.
+3. *Person List*: The list of persons stored in FINDvisor. Note that this list will display the latest list when a command to update the list is **executed successfully**.
+4. *Today's Meeting List*: The list of persons who have a meeting on the current date of your computer when running the application (if any). Note that this list will display the latest list when a command to update the list is **executed successfully**.
```
Comment (KohGuanZeh) [inline docs/UserGuide.md:54]:
```suggestion
4. *Today's Meeting List*: The list of persons who have a meeting on the current date of your computer when running the application (if any). This list is updated accordingly to changes for all persons information.
```
Would this be better? I think the part about `when a command to update the list is **executed successfully**` can be phrased better. What do the rest think @Dethada @Arixeyeion @YuZizhen ?

Comment (Arixeyeion) [reply docs/UserGuide.md:54]: I agree with the change, but I would like to propose a small tweak.

"This list is updated according to changes made for all persons information when a command is executed successfully, in the event it adds, modifies or deletes a person with a meeting on the present day"

Comment (Javiery3889) [reply docs/UserGuide.md:54]: @KohGuanZeh @Arixeyeion
How about this:
> 4. *Today's Meeting List*: The list of persons who have a meeting on the current date of your computer when running the application (if any). This list is updated accordingly to match the latest persons' infromation in *Person List*.

**Review by KohGuanZeh — APPROVED** [review]
LGTM

**Review by Arixeyeion — APPROVED** [review]

**Pull request merged**